src/components/Intro/MucIntro.vue

```diff
@@ -1,9 +1,19 @@
 <script setup lang="ts">
 defineProps({
+  /**
+   * Title of the Intro
+   */
   title: {
     type: String,
   },
 });
```

Comment:
```suggestion
defineProps<{
  /**
   * Title of the Intro
   */
  title: string;
}>();
```
Same as above, using the `pure type` variant.